ultralytics/utils/loss.py

```diff
@@ -301,9 +301,11 @@ def __init__(self, model):  # model must be de-paralleled
         super().__init__(model)
         self.kpt_shape = model.model[-1].kpt_shape
         self.bce_pose = nn.BCEWithLogitsLoss()
-        is_pose = self.kpt_shape == [17, 3]
+        OKS_SIGMA = model.args.OKS_SIGMA  # Sigma values read from cfg
         nkpt = self.kpt_shape[0]  # number of keypoints
-        sigmas = torch.from_numpy(OKS_SIGMA).to(self.device) if is_pose else torch.ones(nkpt, device=self.device) / nkpt
+        # use 1/nkpt as default sigma value if shapes do not match.
+        sigmas = torch.from_numpy(np.array(OKS_SIGMA)).to(
```

Comment:
I think we can just use torch.tensor(OKS_SIGMA) instead of torch.from_numpy(np.array()) right?